docs/formatting.rst

```diff
@@ -0,0 +1,112 @@
+.. _formatting:
+.. currentmodule:: pint
+
+
+.. ipython:: python
+   :suppress:
+
+   import pint
+
+
+String formatting
+=================
+The conversion of :py:class:`Unit` and :py:class:`Quantity` objects to strings (e.g.
+through the :py:class:`str` builtin or f-strings) can be customized using :ref:`format
+specifications <formatspec>`. The basic format is:
+
+
+.. code::
+
+   [magnitude format][modifier][unit format]
```

Comment:
it appears that this is invisible after building

Reply:
well, that's weird. I can not reproduce locally, so this might have something to do with RTD. In any case, I worked around this using the `::`-at-the end-of-a-text-block syntax (the only disadvantage is that it has syntax highlighting)